### PR TITLE
Fix macro being wrong for lookup file directory

### PR DESCRIPTION
**Pull request by ConorMFinn**
### Description of work
Fix as the dev machines can use the instrument name as it's the same as the host. This is different on the NDXs 

Fixes ISISComputingGroup/IBEX#7415

---

#### Code Review

- [ ] A copy of the manual has been placed on the shared drive
- [ ] Pertitent information has been stored in the [wiki](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Specific-Device-IOC)
- [ ] Does the IOC conform to IBEX standards?
    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)
- [ ] If an OPI has been modified, does it conform to the [style guidelines](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/OPI-Creation)?
- [ ] Do the IOC and support module conform to the new [build guidelines](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Reducing-Build-Dependencies)
- [ ] Have the changes been recorded appropriately in a PR for [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)?
- [ ] Is the device's flow control setting correct? [For most devices flow control should be OFF](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Flow-control).

### Functional Tests

- IOC responds correctly in:
    - [ ] Devsim mode
    - [ ] Recsim mode
    - [ ] Real device, if available
- [ ] Supplementary IOCs (`..._0n` where `n>1`) run correctly
- [ ] Log files do not report undefined macros (serach for `macLib: macro` to find instances of `macLib: macro [macro name] is undefined...`

### Final steps

- [ ] Update the IOC submodule in the main EPICS repo. See [Git workflow](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Git-workflow) page for details.
- [ ] Reviewer has merged the associated PR for the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)

### CODE REVIEW

File: MERCURY_ITC/iocBoot/iocMERCURY-IOC-01/st-temp-spc.cmd [hunk 1]
```diff
@@ -4,7 +4,7 @@
 stringiftest("FLOW_SPC_PRESSURE_$(TEMP_NUM)", "$(SPC_TYPE_$(TEMP_NUM)=NONE)", 5, "FLOW")
 stringiftest("VTI_SPC_PRESSURE_$(TEMP_NUM)", "$(SPC_TYPE_$(TEMP_NUM)=NONE)", 5, "VTI")
 
-epicsEnvSet "VTI_CALIB_BASE_DIR" "$(VTI_CALIB_BASE_DIR=$(ICPCONFIGBASE)/$(INSTRUMENT))"
+epicsEnvSet "VTI_CALIB_BASE_DIR" "$(VTI_CALIB_BASE_DIR=$(ICPCONFIGBASE)/$(ICPCONFIGHOST))"
```
Comment (rerpha) [inline MERCURY_ITC/iocBoot/iocMERCURY-IOC-01/st-temp-spc.cmd:7]: are you trying to look in `C:\instrument\settings\config\<>\configurations\` ? if so i think you can remove `$(ICPCONFIGBASE)/$(ICPCONFIGHOST)` and replace with `$(ICPCONFIGROOT)` a bit like https://github.com/ISISComputingGroup/EPICS-ioc/blob/master/TC/iocBoot/iocTC-IOC-01/st-common.cmd#L7

Comment (rerpha) [reply MERCURY_ITC/iocBoot/iocMERCURY-IOC-01/st-temp-spc.cmd:7]: ignore me, im being dumb, i guess you are looking in common instead

Comment (ConorMFinn) [reply MERCURY_ITC/iocBoot/iocMERCURY-IOC-01/st-temp-spc.cmd:7]: So the path we're actually aiming for is `C:\Instrument\Settings\config\<INST_NDX>\calib\mercury_vti_spc`

Comment (rerpha) [reply MERCURY_ITC/iocBoot/iocMERCURY-IOC-01/st-temp-spc.cmd:7]: `$(ICPCONFIGROOT)` gets you to `C:\instrument\settings\config\<>\configurations\` so you'd have to `..` - i think i prefer your current solution

**Review by rerpha — APPROVED** [review]

**Pull request merged**